app/js/controllers/main.js

```diff
@@ -84,6 +107,8 @@ export default class MainController extends BaseController {
         break;
       }
     }
+
+    this[p.initVoiced]();
```

Comment:
This allows to set the hash "voiced" also when the app is running, no need to reload anymore !

I noticed that "hashChanged" is called twice at startup: once when we clear the hash in main() (https://github.com/fxbox/calendar/pull/42/files#diff-67d03633c2ddd4ab1476f82322f28059L60) and once when we set it to `reminders` or `users/login` just after.